src/pages/ReviewPage.jsx

```diff
@@ -0,0 +1,55 @@
+import {useParams} from "react-router-dom"
+import {useState, useEffect} from "react"
+
+
+export default function ReviewPage() {
+    const {review_id} = useParams()
+    const [review, setReview] = useState({})
+    const [reviewComments, setReviewComments] = useState([])
+    const [err, setErr] = useState()
+
+    useEffect(() => {
```

Comment:
youre not using isLoading here but kudos on getting ahead of the game in terms of your error handling